css/editEvents.less

```diff
@@ -0,0 +1,89 @@
+#expandCollapseAllEvents {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+/* Copied from http://jsfiddle.net/KyleMit/Wc4xt/ */
+.panel-heading a:after {
+  font-family: 'Glyphicons Halflings';
+  content: "\e114"; /* glyphicon-chevron-down */
+  float: right;
+  color: grey;
```

Comment:
This is a great place to learn how to use mixins! Any class can be used as a mixin. You can just call .glyphicon and .glyphicon-chevron-down here to give the font family, content, etc. When you call a class as a mixin like this, all the rules that are defined in those classes get output into the class here.

See: http://lesscss.org/features/#mixins-feature